library/core/src/num/f32.rs

```diff
@@ -689,6 +692,9 @@ impl f32 {
 
     /// Returns the minimum of the two numbers.
     ///
+    /// Follows the IEEE-754 2008 semantics for minNum, except for handling of signaling NaNs.
+    /// This match’s the behavior of libm’s fmin.
+    ///
```

Comment:
The PDF explains something nicely, something similar could be added to the min/max notes to explain better the purpose of minimum/maximum:

```
min(1.0, min(1.0, sNaN)) -> min(1.0, qNaN) -> 1.0
min(min(1.0, 1.0), sNaN) -> min(1.0, sNaN) -> qNaN
```
 
With this non-associativity, different compilations or runs on parallel processing can return different answers, depending on whether the sNaN is encountered last or not in a sequence of operations.

Reply:
Adding this would be tricky, because of the behavior of `libm`'s `fmin`. See "3.1 C11: qNaN Are Missing Data" in the pdf:

> The fmax functions determine the maximum numeric value of their arguments.242) 
> The fmin functions determine the minimum numeric value of their arguments.243) 
> [...]
> 242) NaN arguments are treated as missing data: if one argument is a NaN and the other numeric, then the fmax functions **choose the numeric value**.
> 243) The fmin functions are analogous to the fmax functions in their treatment of NaNs.

Witch means:
> fmax (1, qNaN) -> 1
> fmax (1, sNaN) -> undefined   // 1 with glibc libm